modules/react/checkbox/lib/CheckboxInput.tsx

```diff
@@ -65,41 +65,41 @@ const checkboxInputStencil = createStencil({
     marginTop: calc.negate(px2rem(3)),
     marginInlineStart: calc.negate(px2rem(3)),
     position: 'absolute',
-    opacity: 0,
+    opacity: system.opacity.zero,
 
     '&:not(:disabled)': {
       cursor: 'pointer',
     },
 
-    '&:where(:hover, .hover) ~ span:first-of-type': {
+    '&:where(:hover,.hover) ~ span:first-of-type': {
       boxShadow: `0 0 0 ${px2rem(7)} ${base.soap200}`,
```

Comment:
There is no `fg` color for `soap200`. Should I use `system.color.bg.alt.soft` instead?